modules/swagger-codegen/src/main/java/io/swagger/codegen/languages/AbstractKotlinCodegen.java

```diff
@@ -460,6 +460,12 @@ public String toModelName(final String name) {
         return titleCase(modifiedName);
     }
 
+    @Override
+    public String toOperationId(String operationId) {
+        String superOperationId = super.toOperationId(operationId);
+        return camelize(sanitizeName(superOperationId), true);
```

Comment:
rather than just `sanitizeName`, this should also wrap in `sanitizeKotlinSpecificNames`:

```
return camelize(sanitizeKotlinSpecificNames(sanitizeName(superOperationId)), true);
```

Example operation ids that should generate invalid code without `sanitizeKotlinSpecificNames`:

* `_`
* `__`
* `___`
* `1st`
* `123`
* `14710C3C-70B7-4DCC-A529-DBB6BD004D1D`

Maybe could you include these with your tests against this method?  All of the above are valid Swagger operation ids ([see OpenAPI 2.0 docs](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md#operation-object)):

>  Unique string used to identify the operation. The id MUST be unique among all operations described in the API. Tools and libraries MAY use the operationId to uniquely identify an operation, therefore, it is recommended to follow common programming naming conventions.

It's not unheard of for APIs to use UUIDs for operation ids (I think fitbit's docs used to do this).